storage/sql/migrations/updates/20190204170819_migrate_trs_dependant_tables_to_trs_asset_field.sql

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+
+/*
+  DESCRIPTION: Migrate all trs dependant tables into trs asset field
+
+  PARAMETERS: None
+*/
+
+-- Migrate transfer table into trs transferData field
+UPDATE trs
```

Comment:
Please move 

> ALTER TABLE "trs" ADD COLUMN IF NOT EXISTS "asset" jsonb; 

to this migration and at the end of migration create the index on asset.